### ISSUE TITLE
Video download (api.daniil.it) offline for 2+ weeks

### ISSUE THREAD

**Issue by davemachado** [toddmotto/public-apis#307]
```
Issues :-(
> Links 
  1. [L466] 521 https://api.daniil.it  
> Dupes 
  None ✓
```

Currently we have an API that has been offline for 2+ weeks. Looking to discuss if we should remove this to help save the build and our sanity.

**Comment by davemachado** [comment]
I'll leave this open for another ~24 hours for any last minute pitchforks and torches and if no angry mob shows up, I'll remove the dead endpoint and close this issue.

**Comment by davemachado** [comment]
Resolved in commit a891a862913f0e3e54441bab7c641b4abff774a6
